Content.Server/Light/EntitySystems/EmergencyLightSystem.cs

```diff
@@ -192,10 +203,30 @@ private void TurnOff(EntityUid uid, EmergencyLightComponent component)
         _ambient.SetAmbience(uid, false);
     }
 
+    // Turn off and set color
+    private void TurnOff(EntityUid uid, EmergencyLightComponent component, Color color)
+    {
+        _pointLight.SetEnabled(uid, false);
+        _pointLight.SetColor(uid, color);
+        _appearance.SetData(uid, EmergencyLightVisuals.Color, color);
+        _appearance.SetData(uid, EmergencyLightVisuals.On, false);
+        _ambient.SetAmbience(uid, false);
+    }
+
```

Comment:
Use doc summary + Entity<T>

Reply:
added the doc summary, not sure what you meant by the entity part

Reply:
use a single `Entity<EmergencyLightComponent>` parameter instead of two parameters

Reply:
Fixed!  Dear lord I was horrified by this requested change lol but it was easier than I had thought once I understood how it worked